requirements.txt

```diff
@@ -1,5 +1,5 @@
 numpy
-scipy
+scipy<1.7.0
```

Comment:
what is motivating this? We generally want to be using the most up-to-date version of scipy we possibly can

Reply:
I actually found that the latest version of scipy seemed to be bugging out when I was testing. test_gen_plan_params (tests.Prototypes.test_PlanetPopulation.TestPlanetPopulation) had the following error:

ValueError: For each axis slice, the sum of the observed frequencies must agree with the sum of the expected frequencies to a relative tolerance of 1e-08, but the percent differences are
8.378684535698255e-05

I can work on seeing if I can resolve the issue with the latest version of scipy.